stable/atlantis/values.yaml

```diff
@@ -0,0 +1,131 @@
+## -------------------------- ##
+# Values to override for your instance.
+## -------------------------- ##
+
+## An option to override the atlantis url,
+##   if not using an ingress, set it to the external IP.
+# atlantisUrl: http://10.0.0.0
+
+# Replace this with your own repo whitelist:
+orgWhitelist: github.com/yourorg/*
+
+# If using GitHub, specify like the following:
+# github:
+#   user: foo
+#   token: bar
+#   secret: baz
+# GitHub Enterprise only:
+#   hostname: github.your.org
+# (The chart will perform the base64 encoding for you for values that are stored in secrets.)
+
+# If using GitLab, specify like the following:
+# gitlab:
+#   user: foo
+#   token: bar
+#   secret: baz
+# GitLab Enterprise only:
+#   hostname: gitlab.your.org
+# (The chart will perform the base64 encoding for you for values that are stored in secrets.)
+
+# If using Bitbucket, specify like the following:
+# bitbucket:
+#   user: foo
+#   token: bar
+# Bitbucket Server only:
+#   secret: baz
+#   base_url: https://bitbucket.yourorganization.com
+# (The chart will perform the base64 encoding for you for values that are stored in secrets.)
+
+
+# When referencing Terraform modules in private repositories, it may be helpful
+# (necessary?) to use redirection in a .gitconfig like so:
+# gitconfig:
+  # [url "https://YOUR_GH_TOKEN@github.com"]
+  #   insteadOf = https://github.com
+  # [url "https://YOUR_GH_TOKEN@github.com"]
+  #   insteadOf = ssh://git@github.com
+  # [url "https://oauth2:YOUR_GITLAB_TOKEN@gitlab.com"]
+  #   insteadOf = https://gitlab.com
+  # [url "https://oauth2:YOUR_GITLAB_TOKEN@gitlab.com"]
+  #   insteadOf = ssh://git@gitlab.com
+# Source: https://stackoverflow.com/questions/42148841/github-clone-with-oauth-access-token
+
+# To specify AWS credentials to be mapped to ~/.aws:
+# aws:
+#   credentials: |
+#     [default]
+#     aws_access_key_id=YOUR_ACCESS_KEY_ID
+#     aws_secret_access_key=YOUR_SECRET_ACCESS_KEY
+#     region=us-east-1
+#   config: |
+#     [profile a_role_to_assume]
+#     role_arn = arn:aws:iam::123456789:role/service-role/roleToAssume
+#     source_profile = default
+
+## To be used for mounting credential files (when using google provider).
+serviceAccountSecrets:
+  # credentials: <json file as base64 encoded string>
+  # credentials-staging: <json file as base64 encoded string>
+
+
+## -------------------------- ##
+# Default values for atlantis (override as needed).
+## -------------------------- ##
+
+image:
+  repository: runatlantis/atlantis
+  tag: v0.4.10
+  pullPolicy: IfNotPresent
+
+## enable using atlantis.yaml file
+allowRepoConfig: false
+
+# We only need to check every 60s since Atlantis is not a high-throughput service.
+livenessProbe:
+  enabled: true
+  periodSeconds: 60
+  initialDelaySeconds: 5
+  timeoutSeconds: 5
+  successThreshold: 1
+  failureThreshold: 5
+  scheme: HTTP
+readinessProbe:
+  enabled: true
+  periodSeconds: 60
+  initialDelaySeconds: 30
```

Comment:
Agreed, the default is fine since Atlantis starts in <1s. The periodSeconds is set to 60 just because Atlantis isn't high throughput so there's no point hammering it with health checks

Reply:
Changed to 5s.